.azure/templates/steps/system_test_general.yaml

```diff
@@ -97,6 +97,16 @@ jobs:
       displayName: "Setup environment for upgrade"
       condition: contains('${{ parameters.profile }}', 'upgrade')
 
+    - bash: |
```

Comment:
The description talks about _possibilities_ and Microshift. But you seem to use it here in the regular STs. I don't have a problem with that as long as the Connect Build is tested as well. Btu the description and Pr name should make it more clear I think.

Reply:
Yeah I think we discussed it before that it would be good to have the image for Connect tests pre-built and don't rely on the Connect build (it will decrease the execution time). I will change the name of the PR and description. Thanks

Reply:
PR name and description updated